packages/bot-api/src/MessageHandler.ts

```diff
@@ -49,10 +49,9 @@ export abstract class MessageHandler {
 
   async addUser(conversationId: QualifiedId, userId: string) {
     if (this.account?.service) {
-      await this.account.service.conversation.addUsers({
+      await this.account.service.conversation.addUsersToProteusConversation({
         conversationId,
-        userIds: [{domain: '', id: userId}],
-        protocol: ConversationProtocol.PROTEUS,
+        qualifiedUserIds: [{domain: '', id: userId}],
```

Comment:
question, why are we stripping off the qualifiedId that we pass in here? seems the domain will be relevant if it exists

Reply:
ive noticed this in a few other spots recently, wondering if it could be causing any inconsistencies